module_utils/ns1.py

```diff
@@ -29,7 +29,7 @@ def missing_required_lib(msg, reason=None, url=None):
 NS1_COMMON_ARGS = dict(
     apiKey=dict(required=True, no_log=True),
     endpoint=dict(required=False, type="str", default=None),
-    ignore_ssl=dict(required=False, type="bool", default=None),
+    ignore_ssl=dict(required=False, type="bool", default=False),
```

Comment:
I'm not against changing this default value, just wondering why you prefer to do it here rather than let the SDK pick it?

Reply:
I just don't like triple-value for booleans. Passthrough is nice tho, and consistent with the endpoint arg, so I'm gonna revert and add a comment.